.github/workflows/ci.yml

```diff
@@ -2,9 +2,9 @@ name: CI
 
 on:
   push:
-    branches: [main]
+    branches: [main, rjsf-v6]
   pull_request:
-    branches: [main]
+    branches: [main, rjsf-v6]
```

Comment:
You can revert these changes now that the `rjsf-v6` branch is merged and deleted